PyPDF2/_utils.py

```diff
@@ -177,8 +177,6 @@ def read_block_backwards(stream: StreamType, to_read: int) -> bytes:
     read = stream.read(to_read)
     # Seek to the start of the block we read after reading it.
     stream.seek(-to_read, SEEK_CUR)
```

Comment:
Should the comment about "The stream's position should be unchanged" be removed since this is changing the final position of the stream before returning?

Reply:
Yes! Thank you for the hint :hugs: